study.md

```diff
@@ -25,7 +25,7 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, define migrations and explain why developers use them.
 
 ```md
-<!-- your response here -->
+Migrations are for manipulating the database in an MVC structure. They are used for making small and incremental changes to a database, so the database does not get overwhelmed with content.
```

Comment:
The parts about MVC structure and the database getting "overwhelmed" are irrelevant, but the rest is correct.